command/acl/acl_helpers.go

```diff
@@ -45,9 +45,13 @@ func GetTokenAccessorIDFromPartial(client *api.Client, partialAccessorID string)
 }
 
 func GetPolicyIDFromPartial(client *api.Client, partialID string) (string, error) {
-	if partialID == "global-management" {
-		return structs.ACLPolicyGlobalManagementID, nil
+	// try the builtin policies (by name) first
+	for _, policy := range structs.ACLBuiltinPolicies {
```

Comment:
The map is actually ID -> policy, not Name -> policy, which is what it was doing before. But I added an extra line to do a map lookup in case the partialID is actually an ID and we don't have to list them